src/cpp/core/include/core/collection/Position.hpp

```diff
@@ -16,6 +16,7 @@
 #define CORE_COLLECTION_POSITION_HPP
 
 #include <iostream>
+#include <sstream>
```

Comment:
This seems like a stray change given no other changes in the file. Should it be added to `SessionRParser.cpp` instead?

Reply:
Evidently we're using `stringstream` in some inline functions here:

https://github.com/rstudio/rstudio/blob/eb034f72f6c094c47505a000d57974c32284561b/src/cpp/core/include/core/collection/Position.hpp#L65-L70

so it seemed like an error that the header was omitted out previously. But that code also doesn't _really_ need `stringstream`... 🤷